crates/README.md

```diff
@@ -8,10 +8,14 @@ The intention is that (much) longer term, we might be wanting to expose more of
 * `qiskit-pyext` is the only crate that actually builds a Python C extension library.
```

Comment:
Should the `transpiler` crate be exposed as something other than `qiskit._accelerate` to Python? I feel like we are way beyond accelerators at this point and should have a distinction of which rust crates are being used. Maybe we could have `qiskit._accelerate`, `qiskit._circuit`, and `qiskit._transpiler`. Though I understand it might be confusing since it re-uses the same names, it should be considered as rust has become a bigger part of our codebase.

This would be outside of the scope of this PR but I wanted to bring it up here since we're already organizing.

Reply:
No I don't think it should. This was something we actively did in the past and it caused more headaches than it was worth. Having the single Python entrypoint is desirable for a lot of reasons (you can look at the docs and commits around when pyext was added for the context there).